sechub-sarif/src/test/resources/examples/microsoft/sarif-tutorials/samples/3-Beyond-basics/bad-eval.py

```diff
@@ -0,0 +1,4 @@
+# 3-Beyond-basics/bad-eval.py
+
+expr = input("Expression> ")
+print(eval(expr))
```

Comment:
again, this is a real vulnerability.

Reply:
done: dropped the real example part from test resources